documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
+- Python: No additional dependencies are required for Python projects.
+
+If your project uses a language not supported by Code2Flow for flow map generation, the documentation generation capabilities will still be available, but without the flow map visualization for those specific languages.
+
 ## Prerequisites
 
 Ensure the following tools are installed:
 
 - `bito` : https://github.com/gitbito/CLI
 
-- `mermaidcli` : https://github.com/mermaid-js/mermaid-cli
+- `Code2Flow:` : https://github.com/scottrogowski/code2flow
```

Comment:
**Security Issue**: The introduction of Code2Flow as a dependency without specifying the version can lead to the use of outdated or vulnerable versions of the tool. This can expose the documentation generation tool to various security vulnerabilities inherent in the older versions of Code2Flow. <br> **Fix**: Specify a secure and up-to-date version of Code2Flow in the documentation. This ensures that users are directed to use a version that has been vetted for security vulnerabilities and is considered stable for use. <br> **Code Suggestion**: 
 ```
 + Specify a secure and up-to-date version of Code2Flow in the documentation. This ensures that users are directed to use a version that has been vetted for security vulnerabilities and is considered stable for use.
 ```